### PR TITLE
generated new sdk and change test case

### PR DESCRIPTION
**Pull request by ruiminhu**
# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running `generate.ps1`. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version.
	
	**Note: We have recently updated the PSH module called by `generate.ps1` to emit additional data. This would help reduce/eliminate the Code Verification check error. Please run following command**:

	    `dotnet msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules`

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

Link to swagger pr: 
https://github.com/Azure/azure-rest-api-specs/pull/13664

### CODE REVIEW

File: sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/ISignalROperations.cs [hunk 1]
```diff
@@ -47,7 +47,7 @@ public partial interface ISignalROperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<NameAvailability>> CheckNameAvailabilityWithHttpMessagesAsync(string location, NameAvailabilityParameters parameters = default(NameAvailabilityParameters), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<NameAvailability>> CheckNameAvailabilityWithHttpMessagesAsync(string location, NameAvailabilityParameters parameters, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```
Comment (markcowl) [inline sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/ISignalROperations.cs:50]: Breaking change

File: sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/ISignalRPrivateEndpointConnectionsOperations.cs [hunk 2]
```diff
@@ -89,22 +112,20 @@ public partial interface ISignalRPrivateEndpointConnectionsOperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<PrivateEndpointConnection>> UpdateWithHttpMessagesAsync(string privateEndpointConnectionName, string resourceGroupName, string resourceName, PrivateEndpointConnection parameters = default(PrivateEndpointConnection), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<PrivateEndpointConnection>> UpdateWithHttpMessagesAsync(string privateEndpointConnectionName, PrivateEndpointConnection parameters, string resourceGroupName, string resourceName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```
Comment (markcowl) [inline sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/ISignalRPrivateEndpointConnectionsOperations.cs:115]: breaking change

File: sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/Models/SignalRResource.cs [hunk 3]
```diff
@@ -39,70 +39,71 @@ public SignalRResource()
...
+        /// modification of the resource.</param>
+        public SignalRResource(string id = default(string), string name = default(string), string type = default(string), string location = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), ResourceSku sku = default(ResourceSku), string provisioningState = default(string), string externalIP = default(string), string hostName = default(string), int? publicPort = default(int?), int? serverPort = default(int?), string version = default(string), IList<PrivateEndpointConnection> privateEndpointConnections = default(IList<PrivateEndpointConnection>), IList<SharedPrivateLinkResource> sharedPrivateLinkResources = default(IList<SharedPrivateLinkResource>), SignalRTlsSettings tls = default(SignalRTlsSettings), IList<SignalRFeature> features = default(IList<SignalRFeature>), SignalRCorsSettings cors = default(SignalRCorsSettings), ServerlessUpstreamSettings upstream = default(ServerlessUpstreamSettings), SignalRNetworkACLs networkACLs = default(SignalRNetworkACLs), string kind = default(string), ManagedIdentity identity = default(ManagedIdentity), SystemData systemData = default(SystemData))
```
Comment (markcowl) [inline sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/Models/SignalRResource.cs:87]: breaking change

File: sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/Models/SignalRResource.cs [hunk 4]
```diff
@@ -118,40 +119,6 @@ public SignalRResource(string id = default(string), string name = default(string
...
-        /// operation will not be modified.
-        /// And the response will only include featureFlags that are explicitly
-        /// set.
-        /// When a featureFlag is not explicitly set, SignalR service will use
-        /// its globally default value.
-        /// But keep in mind, the default value doesn't mean "false". It varies
-        /// in terms of different FeatureFlags.
-        /// </summary>
-        [JsonProperty(PropertyName = "properties.features")]
-        public IList<SignalRFeature> Features { get; set; }
```
Comment (markcowl) [inline sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/Models/SignalRResource.cs:134]: breaking changes

File: sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/SignalROperations.cs [hunk 5]
```diff
@@ -80,12 +80,16 @@ internal SignalROperations(SignalRManagementClient client)
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<NameAvailability>> CheckNameAvailabilityWithHttpMessagesAsync(string location, NameAvailabilityParameters parameters = default(NameAvailabilityParameters), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<NameAvailability>> CheckNameAvailabilityWithHttpMessagesAsync(string location, NameAvailabilityParameters parameters, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```
Comment (markcowl) [inline sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/SignalROperations.cs:83]: breaking change

File: sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/SignalROperations.cs [hunk 6]
```diff
@@ -795,41 +799,89 @@ public async Task<AzureOperationResponse<SignalRKeys>> ListKeysWithHttpMessagesA
...
         /// </param>
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        public async Task<AzureOperationResponse<SignalRKeys>> RegenerateKeyWithHttpMessagesAsync(string resourceGroupName, string resourceName, RegenerateKeyParameters parameters = default(RegenerateKeyParameters), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse> DeleteWithHttpMessagesAsync(string resourceGroupName, string resourceName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
         {
             // Send request
-            AzureOperationResponse<SignalRKeys> _response = await BeginRegenerateKeyWithHttpMessagesAsync(resourceGroupName, resourceName, parameters, customHeaders, cancellationToken).ConfigureAwait(false);
+            AzureOperationResponse _response = await BeginDeleteWithHttpMessagesAsync(resourceGroupName, resourceName, customHeaders, cancellationToken).ConfigureAwait(false);
```
Comment (markcowl) [inline sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/SignalROperations.cs:846]: did this operation become lro?

File: sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/SignalROperations.cs [hunk 7]
```diff
@@ -1014,89 +1066,41 @@ public async Task<AzureOperationResponse<SignalRResource>> GetWithHttpMessagesAs
...
-        /// Parameters for the create or update operation
-        /// </param>
-        /// <param name='customHeaders'>
-        /// The headers that will be added to request.
-        /// </param>
-        /// <param name='cancellationToken'>
-        /// The cancellation token.
+        /// Parameter that describes the Regenerate Key Operation.
         /// </param>
-        public async Task<AzureOperationResponse<SignalRResource>> CreateOrUpdateWithHttpMessagesAsync(string resourceGroupName, string resourceName, SignalRResource parameters = default(SignalRResource), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```
Comment (markcowl) [inline sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/SignalROperations.cs:1035]: breaking changwe

**Review by markcowl — CHANGES_REQUESTED** [review]
Noted some places where there are breaking changes from the previous api-version.  If the previous version was a preview this may be ok, also please:

- provide a link to the swagger pr
- update the csproj and Properties\AssemblyInfo.cs with the appropriate new version, and include release notes

**Comment by markcowl** [comment]
@ruiminhu Please respond to comments.  If there is no response, will have to close this PR at the end of the week.

File: sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/ISignalROperations.cs (hunk 1 above)
Comment (ruiminhu) [reply sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/ISignalROperations.cs:50]: Hey Markcowl, thanks for the cr, csproj and assembly file is modified. please help take a look

File: sdk/signalr/Microsoft.Azure.Management.SignalR/src/Microsoft.Azure.Management.SignalR.csproj [hunk 8]
```diff
@@ -6,10 +6,10 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.SignalR</PackageId>
 		<Description>Provides management capabilities for Azure SignalR.</Description>
-		<Version>1.1.1-preview</Version>
+		<Version>1.1.2-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.SignalR</AssemblyName>
 		<PackageTags>Microsoft Azure SignalR;SignalR;</PackageTags>
-		<PackageReleaseNotes>This is a public preview release of the Azure SignalR management SDK. Included with this release are SignalR Client certificate and Managed Identity support.</PackageReleaseNotes>
+		<PackageReleaseNotes>This is a public preview release of the Azure SignalR management SDK. Included with this release are SignalR Private End Point Connection support and some fix.</PackageReleaseNotes>
```
Comment (markcowl) [inline sdk/signalr/Microsoft.Azure.Management.SignalR/src/Microsoft.Azure.Management.SignalR.csproj:12]:
```suggestion
		<PackageReleaseNotes>This is a public preview release of the Azure SignalR management SDK. Included with this release are SignalR Private End Point Connection support and some fixes.</PackageReleaseNotes>
```

**Review by markcowl — CHANGES_REQUESTED** [review]
One minor fix, otherwise, this looks good.

**Comment by markcowl** [comment]
@ruiminhu One minor change, otherwise this is ready to go.

File: sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/SignalROperations.cs (hunk 6 above)
Comment (ruiminhu) [reply sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/SignalROperations.cs:846]: sorry，whats lro stand for?

File: sdk/signalr/Microsoft.Azure.Management.SignalR/src/Microsoft.Azure.Management.SignalR.csproj (hunk 8 above)
Comment (ruiminhu) [reply sdk/signalr/Microsoft.Azure.Management.SignalR/src/Microsoft.Azure.Management.SignalR.csproj]: good catch!

File: sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/SignalROperations.cs (hunk 6 above)
Comment (markcowl) [reply sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/SignalROperations.cs:846]: long-running operation.  It looks like this is a bad diff.

**Review by markcowl — APPROVED** [review]

**Pull request merged**